code/names.dm

```diff
@@ -3,6 +3,7 @@ var/list/wizard_first = file2list("config/names/wizardfirst.txt")
 var/list/wizard_second = file2list("config/names/wizardsecond.txt")
 var/list/ninja_titles = file2list("config/names/ninjatitle.txt")
 var/list/ninja_names = file2list("config/names/ninjaname.txt")
+var/list/planet_names = file2list("config/names/planetname.txt")
```

Comment:
Did you include this file?

Reply:
I'm looking for the FTL13 code, as they had a list of planet names

Reply:
There's a star list, but the planets in FTL13 were either uniquely named (Capital planets) or based off of the star (So you'd have Tartarus 3, for instance)